### PR TITLE
Minor cleanup of setup.py and removal of pyall dependency.

### PR DESCRIPTION
**Pull request by lachlanhurst**
Some fairly minor changes just to step through the process of reviewing a pull request.

### CODE REVIEW

**Review by wenjunwu — APPROVED** [review]
all good to me.

File: hyo2/mate/lib/scan_ALL.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 from hyo2.mate.lib.scan import *
```
Comment (giumas) [inline hyo2/mate/lib/scan_ALL.py:1]: It has nothing to do with this PR, but `import *` is commonly known as bad practice: https://stackoverflow.com/questions/2386714/why-is-import-bad

**Review by giumas — APPROVED** [review]
It has nothing to do with this PR, but `import *` is commonly known as bad practice: https://stackoverflow.com/questions/2386714/why-is-import-bad

File: hyo2/mate/lib/scan_ALL.py (hunk 1 above)
Comment (lachlanhurst) [reply hyo2/mate/lib/scan_ALL.py:1]: Yes, agreed.

As I work through the code I'll tidy up these things as I find them.

**Pull request merged**